tests/python/unittest/test_operator.py

```diff
@@ -8054,6 +8054,68 @@ def test_image_normalize():
     # check backward using finite difference
     check_numeric_gradient(img_norm_sym, [data_in_4d], atol=0.001)
 
+@with_seed()
```

Comment:
Could you define a function called `test_index_array` that holds all the test functions you defined here? It's easier and cleaner to maintain just one test function for one operator.

Reply:
Sure! The relevant commit is 3a430daaba4ea04bae90e2326e66febfef70a00f.